src/waveform/waveformwidgetfactory.cpp

```diff
@@ -359,6 +364,16 @@ void WaveformWidgetFactory::addTimerListener(QWidget* pWidget) {
             Qt::DirectConnection);
 }
 
+
+void WaveformWidgetFactory::onSkinLoadFinished() {
+    // This regenerates twice the waveforms because of a bug found on Windows
+    // where the first one fails.
+    // The problem is that the window of the widget thinks that it is not exposed.
+    // (https://doc.qt.io/qt-5/qwindow.html#exposeEvent )
```

Comment:
```suggestion
    // This regenerates the waveforms twice because of a bug found on Windows
    // where the first one fails.
    // The problem is that the window of the widget thinks that it is not exposed.
    // (https://doc.qt.io/qt-5/qwindow.html#exposeEvent )
    // TODO: Remove this when it has been fixed upstream.
```